web-test-runner.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
Could you please add the license header here too? We need it for all files

Reply:
Added